LunaMary/src/SelectedQuiz.java

```diff
@@ -0,0 +1,70 @@
+import java.util.List;
+
+
+class SelectedQuiz implements IQuestion {
```

Comment:
I don't think the name here is correct. Because a Quiz references a group of questions and this implements IQuestion interface which indicates that this class should have a different name, something like:
MultipleChoiceQuestion